dags/resources/stages/enrich/sqls_redshift/logs.sql

```diff
@@ -0,0 +1,33 @@
+INSERT INTO {{params.schema}}.{{params.table}}
+(
+  chain_id,
+  log_index,
+  transaction_hash,
+  transaction_index,
+  address,
+  data,
+  topic0,
+  topic1,
+  topic2,
+  topic3,
+  block_number,
+  block_hash,
+  block_timestamp
+)
+SELECT
+    1 as chain_id,
```

Comment:
once [this](https://github.com/BitskiCo/ethereum-etl/pull/15) is tested and merged, we can use chain_id from export. But need to re-run backfill. So hard coding for now

Reply:
might need to set a flag to `is_set_chain_id` to change the sql. saving that for later when we expand batch job to other chains